tests/codes/test_python_occ.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-FileCopyrightText: 2021-present M. Coleman, J. Cook, F. Franza
+# SPDX-FileCopyrightText: 2021-present I.A. Maione, S. McIntosh
+# SPDX-FileCopyrightText: 2021-present J. Morris, D. Short
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+from copy import deepcopy
+
+import pytest
+
+from bluemira.codes import python_occ
+from bluemira.geometry.face import BluemiraFace
+from bluemira.geometry.tools import extrude_shape, make_polygon
+
+
+class TestImprintSolids:
+    @pytest.mark.parametrize(
```

Comment:
will probably need to add a skip if import not available, we do it for a few other tests with openmc polyscope and process

https://docs.pytest.org/en/4.6.x/skipping.html#skipping-on-a-missing-import-dependency